integration/test/alert/alert.go

```diff
@@ -38,7 +38,8 @@ func GetAlerts() (map[string]int, int) {
 	alertsData := make(map[string]int)
 	totalAlerts := 0
 
-	time.Sleep(3 * time.Minute)
+	duration, _ := time.ParseDuration("3m15s")
```

Comment:
Yes, we do call it twice in case of bookend ems testing. This is by design.

For bookend case,
1. Generate all issuing ems in cluster
2. wait exact 3m(now 3m15s)
3. get data from prom API and cache count
4. Generate all resolving ems in cluster
5. wait exact 3m(now 3m15s)
6. get data from prom API and cache count
7. compare and evaluate that all bookend ems got successfully resolved by second set of ems(step 4).


To add some delay between 2 back to back PRs --> For back to back PR, this ems testing is only happening on native, so it can't be run 2 ems testing at the same time.